frontend/src/pages/Report.tsx

```diff
@@ -835,23 +835,21 @@ export const Report = () => {
         </main>
         <div className="footer">
           <section className="footer-container">
-            <div className="col-8">
-              <QuickAdd
-                addIssueActivity={addIssueActivityHandler}
-                toastList={toastList}
-                onToastListUpdate={handleToastListUpdate}
-                issueInputRef={issueInputRef}
-              ></QuickAdd>
-            </div>
+            <QuickAdd
+              addIssueActivity={addIssueActivityHandler}
+              toastList={toastList}
+              onToastListUpdate={handleToastListUpdate}
+              issueInputRef={issueInputRef}
+            ></QuickAdd>
             {toastList.length > 0 && (
               <Toast onCloseToast={handleCloseToast} toastList={toastList} />
             )}
-            <div className="col-4 save-changes">
-              <div className="unsaved-alert-p">
-                {showUnsavedWarning && (
-                  <p role="status">⚠ You have unsaved changes</p>
-                )}
-              </div>
```

Comment:
When adding time in the sheet the footer elements move up:
![image](https://user-images.githubusercontent.com/105041852/189071344-303f0796-6ffe-43c1-85c6-6b60fd1836a6.png)
![image](https://user-images.githubusercontent.com/105041852/189071407-93ac835a-6c29-42a0-866c-ad56f9ee5bd0.png)

Reply:
fixed